apps/aecore/lib/aecore/chain/chain_state.ex

```diff
@@ -5,120 +5,129 @@ defmodule Aecore.Chain.ChainState do
   """
 
   alias Aecore.Structures.SignedTx
+  alias Aecore.Structures.SpendTx
+  alias Aecore.Structures.DataTx
   alias Aecore.Structures.Account
   alias Aeutil.Serialization
   alias Aeutil.Bits
 
   require Logger
 
-  @type account_chainstate() :: %{binary() => map()}
+  @typedoc "Public key representing an account"
+  @type pubkey() :: binary()
```

Comment:
`@type pubkey()` is used in multiple places with different names `@type pub_key()` sometimes. Move it to the Wallet worker.